keycloak-services/keycloak.yaml

```diff
@@ -1,5 +1,14 @@
 services:
-- hash: 11eba870465c95fe76698965b05eee7dfca4a55f 
+- hash: 11eba870465c95fe76698965b05eee7dfca4a55f
   name: keycloak-deployment
   path: /openshift/keycloak.app.yaml
   url: https://github.com/fabric8-services/keycloak-deployment
+  environments:
+  - name: production
+    parameters:
+      REPLICAS: 3
+      OPERATING_MODE: clustered
+  - name: staging
+    parameters:
+      REPLICAS: 3
```

Comment:
No, everything is converted to strings by calling subprocess in python, so this is fine